src/webui/api/appcontroller.cpp

```diff
@@ -302,6 +302,8 @@ void AppController::preferencesAction()
     data["send_buffer_watermark"] = session->sendBufferWatermark();
     data["send_buffer_low_watermark"] = session->sendBufferLowWatermark();
     data["send_buffer_watermark_factor"] = session->sendBufferWatermarkFactor();
+    // Outgoing connections per second
+    data["connection_speed"] = session->connectionSpeed();
```

Comment:
This requires increasing API version (unless it's already increased against to previous release).

Reply:
Done.